content/pages/nuttx/getting-started/index.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Getting Started with NuttX and ESP32 Family"
+date: 2024-06-20T16:54:43-03:00
+tags: ["NuttX"]
+authors:
+    - "tiago-medicci"
+    - "eren-terzioglu"
+    - "filipe-cavalcanti"
+---
+
+## Getting Started with NuttX and ESP32 Family
```

Comment:
The title in the YAML header above is already rendered as the first level heading. This line repeats the same title as the second level heading again.

Suggest removing it.

Also, all the headings below can be moved one level up, for example:

```
## Apache NuttX RTOS
```

Reply:
Done!